services/JsonReader.class.php

```diff
@@ -0,0 +1,344 @@
+<?php
+/**
+ *  This is a General JsonReader
+ *  It has the ability to:
+ *      Read .json ext files and return json Object
+ *      Performs Basic -> get/add-set/delete functions On Json Object
+ *
+ *  To get variable of this class use  -> JsonReader::getInstance();
+ *  To Call a method from this class   -> $jr = ^^^^^^^^^^^^^^^; $jr -> ClassFunction();
+ *
+ */
+class JsonReader
+{
+    /**
+     * Json Object Example for Reference
+     *   {
+     *       "data": {
+     *           "_headers": "lname,fname,something1,something2,something3",
+     *           "_last_Modified": "TIMESTAMP",
+     *           "indexes": {
+     *               "lname": 1,
+     *               "fname": 2,
+     *               "something1": 3,
+     *               "something2": 4,
+     *               "something3": 5
+     *           },
+     *           "_last_calculated": "TIMESTAMP"
+     *       }
+     *   }
+     */
+    /**
+     * @param _instance null if class never initilized. Class instance if was before initilized
+     *                  Purpose: To only have one instance of this object no matter how many places it is called.
+     */
+    private static $_instance;
+    //verbose
+    private static $v = false;
+    public function getInstance()
+    {
+        if (self::$_instance == null)
+        {
+            self::$_instance = new self();
+        }
+        return self::$_instance;
+    }
+    /**
+     * This Constructor is PRIVATE. To get class instance call getInstance method
+     */
+    private function __construct()
+    {
+        if (self::$v)
+        {
+            print "<br>" . __CLASS__ . " Constructed<br>";
+        };
+    }
+
+    /**
+     * This function takes any object and converts to Array
+     */
+    function objectToArray($d)

```

Comment:
if this is pretty much finalized, then remove the verbose functionality or atleast comment it